wikitemplate.md

```diff
@@ -96,10 +96,10 @@
 
 - [ ] Verify that none of the reward endpoints are being contacted when a user visits a media publisher (`youtube.com`, `reddit.com`, `twitter.com`, `github.com`) and hasn't interacted with rewards
   - [ ] Verify that `rewards.brave.com`, `pcdn.brave.com`, `grant.rewards.brave.com` or `api.rewards.brave.com` are not being contacted
-- [ ] Verify you are able to create a new wallet.
-- [ ] Verify you are able to restore a wallet.
-- [ ] Verify account balance shows correct BAT and USD value.
-- [ ] Verify actions taken (claiming grant, tipping, auto-contribute) display in wallet panel.
+- [ ] Verify you are able to create a new Rewards profile.
+- [ ] Verify you are able to restore an old Rewards profile.
+- [ ] Verify BAT balance shows correct BAT and USD value.
```

Comment:
Same as above

Reply:
@srirambv can adjust this wording 👍🏻 

will create a follow up.